src/main/webapp/index.jsp

```diff
@@ -13,23 +13,34 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 --%>
+<%
+String[] admins = {"Cynthia", "Justin", "Sergio"};
```

Comment:
Rather than maintain this list in two separate places, why not have a static method that checks `isAdmin` in a separate file and use the output of that method in both the admin controller and as an attribute in your template.

Reply:
By this you mean that I could move the current isAdmin method to another class and then calling it in both the servlet and admin.jsp, right? Do you know how I would go about calling a static method in jsp? I've been searching around and it seems like that I might only be able to call instance methods.

Reply:
Vasu, I've found the problem! In my isAdmin function I should have been calling admin.equals(user) instead of user.equals(admin) because I even said in the docstring that a null user could be passed in... so I got a NullPointerException when opening the server since it immediately puts me on index.jsp. I'll push the new changes right away.